content/en/continuous_integration/intelligent_test_runner/python.md

```diff
@@ -57,8 +57,61 @@ After completing setup, run your tests as you normally do:
 DD_ENV=ci DD_SERVICE=my-python-app DD_CIVISIBILITY_ITR_ENABLED=true pytest --ddtrace
 {{< /code-block >}}
 
+## Preventing the Intelligent Test Runner from skipping specific tests or modules
```

Comment:
I wonder if we can come up with a header that is more succinct. This one seems a bit long for a header. Maybe something along the lines of `Disable skipping for specific tests or modules`?

Reply:
This looks fine to me too!